utils/interfaces/_library/core.py

```diff
@@ -49,7 +49,10 @@ def get_expected_timeout(self, context):
         elif context.library.library in ("golang",):
             result = 10
         elif context.library.library in ("nodejs",):
-            result = 5
+            # if scenario is REMOTE_CONFIG_MOCKED_BACKEND_ASM_FEATURES
+            # result = 60
+            # else result = 5
+            result = 60
```

Comment:
@cbeauchesne I don't know what's the best way to do that but the async timeout should be bigger for this scenario (and maybe a few others).

Reply:
-> https://github.com/DataDog/system-tests/blob/main/utils/interfaces/_profiling.py#L14

Reply:
I tried that and couldn't make it work, it was throwing "can't collect test because it has a constructor", can you please do this for me or document it somehow so I can do it myself ?